frontend/src/views/Abfrage.vue

```diff
@@ -3,9 +3,7 @@
     <DefaultLayout solid-heading>
       <template #content>
         <div v-if="step === 1">
-          <infrastrukturabfrageComponent
-            v-model="abfrage"
-          />
+          <infrastrukturabfrageComponent v-model="abfrage" />
```

Comment:
Wird hier das div benötigt? Kann man das nicht ähnlich zu den anderen Componenten machen?

```suggestion
          <infrastrukturabfrageComponent
            v-if="step === 1"
            v-model="abfrage"
           />
```